### PR TITLE
Add wasm example to manual

### PR DESCRIPTION
**Pull request by hayd**
cc @kevinkassimo

### CODE REVIEW

File: std/manual.md [hunk 1]
```diff
@@ -951,6 +951,51 @@ for await (const req of serve(":8000")) {
 $ deno run --importmap=import_map.json hello_server.ts
 ```
 
+## WASM support
+
+Deno can import a [wasm](https://webassembly.org/) binary.
```
Comment (ry) [inline std/manual.md]: I think we should mention that Deno supports the normal way of loading WASM too 
https://github.com/denoland/deno/blob/master/cli/tests/wasm.ts

Comment (hayd) [reply std/manual.md]: Thanks for fixing.

**Comment by hayd** [comment]
I do actually like the "this is a self contained wasm file we've actually compiled here", not a black box binary, but understand that can be documented elsewhere.

**Comment by ry** [comment]
Building wasm is something we would like to be within the scope of the deno project - but we haven't decide how to address this yet. Until then I would rather leave it with these basic documentation.

**Pull request merged**